src/sagemaker_containers/transformers.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import collections
+
+from sagemaker_containers import env, serializers
+
+TransformSpec = collections.namedtuple('TransformSpec', 'serialized_prediction accept')
+
+
+class BaseTransformer(object):
+    """A Transformer is a proxy between the worker and the framework transformation functions.
+
+    The BaseTransformer implements the following functions:
+
+        functions required by worker.run:
+
+            initialize, transform
+
+        functions required by the frameworks:
+
+            input_fn, output_fn, transform_fn
+
+    Examples:
+    >>>import os
+
+    >>>from sagemaker_containers import env, modules, transformers
+    >>>import miniml
+
+    >>>serving_env = env.ServingEnv()
+
+    >>>class MiniMlTransformer(transformers.BaseTransformer):
```

Comment:
MiniMl?

Reply:
Just a fake example. I will change it to a Keras example